File: fibonacci_numbers/iterative.js
const getFib = (pos) => {
  if (pos == 0 || pos === 1) return pos;
  let first = 0,
    second = 1,
    next = first + second;
  for (let i = 2; i < pos; i++) {
    first = second;
    second = next;
    next = first + second;
  }
  return next;
};

console.log(getFib(1));
console.log(getFib(2));
console.log(getFib(3));
console.log(getFib(4));
console.log(getFib(5));
console.log(getFib(6));

File: fibonacci_numbers/iterative.py
def get_fib(pos):
    if pos == 0: return 0
    if pos == 1: return 1
    first = 0
    second = 1
    next = first + second
    for i in range(2, pos):
        first = second
        second = next
        next = first + second
    return next

print(get_fib(1))
print(get_fib(2))
print(get_fib(3))
print(get_fib(4))
print(get_fib(5))
print(get_fib(6))